packages/apps/reputation-oracle/server/src/modules/payout/payout.service.ts

```diff
@@ -213,6 +215,23 @@ export class PayoutService {
       return accMap;
     }, new Map<string, typeof bountyValue>());
 
+    // Update reputation for each result
+    for (const result of annotations.results) {
```

Comment:
this should not be here, this should be inside reputation service.
We have two cron jobs:
1. Payouts management
2. Reputation management: here is where reputation assesment should happen